src/main/java/org/apache/commons/io/EndianUtils.java

```diff
@@ -78,19 +84,20 @@ public static double readSwappedDouble(final InputStream input) throws IOExcepti
     }
 
     /**
-     * Reads a "float" value from a byte array at a given offset. The value is
-     * converted to the opposed endian system while reading.
+     * Reads a little endian "float" value from a byte array at a given offset.
```

Comment:
I think that using `{@code _type_}` would be better here.

Reply:
Maybe. What's really confusing here is we need to distinguish two different "floats":

1. The Java float primitive in memory in the VM.
2. A 4 byte chunk read from an input stream or an array. 

I'm not sure I've got this distinction clear enough yet. Let me play with it a bit more.